tests/routes/v1/test_plates.py

```diff
@@ -307,21 +304,23 @@ def test_post_plates_endpoint_fit_to_pick_type_with_ss_create_failure(
 def test_post_plates_endpoint_all_samples_type_with_ss_plate_lookup_failure(
     app, client, source_plates, mocked_responses, endpoint, plate_type
 ):
-    mock_plates_lookup_failure(app, mocked_responses)
+    mock_labware_lookup_failure(app, mocked_responses)
 
     body = create_plate_body(VALID_PLATE_BARCODE, plate_type)
 
     response = client.post(endpoint, json=body)
 
     assert response.status_code == HTTPStatus.INTERNAL_SERVER_ERROR
-    assert response.json == CONNECTION_ERROR_JSON
+    assert response.json == {
+        "errors": ["An unexpected error occurred attempting to create a plate in Sequencescape: (AssertionError)"]
+    }
```

Comment:
I was thinking that myself and the AssertionError does have a message component attached to it.  The source of this response message is used for a variety of different possible thrown exceptions so keeping it as just the name of the message type does make some sense to avoid exposing sensitive information about the errors being thrown.  The detailed log of the issue is added to the Lighthouse logger.  This is just what the caller of the Lighthouse API sees coming back.